### ISSUE TITLE
Service paused / unavailable

### ISSUE THREAD

**Issue by caminsha** [stefanjudis/tiny-helpers#560]
Unfortunately, at the moment, I cannot visit the website.

### Actual Behaviour

When I want to visit the website, I get the following error code:
```
503: SERVICE_UNAVAILABLE 
Code: DEPLOYMENT_PAUSED 
ID: fra1::tv5jv-1733606841096-7d2d5c9c18bf
```

Is this a planned pause/maintenance?

#### Expected behaviour

See the website like normal.

**Comment by stefanjudis** [comment]
Yes, TH was cloned, and massive traffic hit the site and infrastructure, which caused a $300 cost on my end. It'll come back online eventually. :)

**Comment by Ryuno-Ki** [comment]
Hope you're fine. That sounds like an awful thing to happen short before the holiday season :weary:

**Comment by stefanjudis** [comment]
@Ryuno-Ki Thank you! Yeah, I'm alright. Thanks for asking. It's just strange. I was thinking "Yeah [this site](https://anamericanvictory.net/home/index.html) can't have that much traffic", and the cost grew and grew. We're talking millions of requests here. 😅

It's definitely on me not to have taken it seriously. I'll bring TH back online soon'ish and have yet to find a cost effective way to "fight back a little" bc they're still using my infra for site screenshots.

**Comment by Ryuno-Ki** [comment]
https://github.com/ai-robots-txt/ai.robots.txt proved helpful on my end :angel:

**Comment by caminsha** [comment]
Thank you, @stefanjudis for the explanation.

This is indeed pretty bad news and totally understandable that you stopped the website for the moment.

**Comment by stefanjudis** [comment]
And we're back online with adjusted firewall settings. :) Thanks for the patience. :D
